### PR TITLE
[2/2] Add new hobbies page

### PR DESCRIPTION
**Pull request by dash102**
Contains hobbies page below the education and work pages.

### CODE REVIEW

**Comment by dash102** [comment]
Completed version of website - alphabetized CSS will come in the next PR after merging the current changes.

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -106,6 +114,31 @@ td p {
   color: rgb(61,95,121);
 }
 
+#hobbies-div {
+  color: white;
+}
+
+.hobbies-page-a {
```
Comment (liangqigong) [inline portfolio/src/main/webapp/style.css:121]: the name 'page-a' is not very meaningful. Can you rename it?

And it seems it is not following alphabetic order.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -106,6 +114,31 @@ td p {
...
+
+.hobbies-description {
+  padding: 3rem 4rem 3rem 1rem;
+}
+
+.hobbies-imgs tr td img {
+  height: 8rem;
+}
+
+.hobbies-image {
```
Comment (liangqigong) [inline portfolio/src/main/webapp/style.css:134]: I don't see where it is used?

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -106,6 +114,31 @@ td p {
...
+  margin: .5rem;
+}
+
+.hobbies-description td {
```
Comment (liangqigong) [inline portfolio/src/main/webapp/style.css:138]: Not following alphabetic order here?

Comment (dash102) [reply portfolio/src/main/webapp/style.css:138]: Could you clarify what should be alphabetized? I was going to do another PR that alphabetized the attributes within each block, but should the blocks themselves be following some order?

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (dash102) [reply portfolio/src/main/webapp/style.css:134]: Ah I must have left that from a previous version, will remove.

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (dash102) [reply portfolio/src/main/webapp/style.css:121]: Will do. 
With alphabetization, color and text-decoration should be in the right order, do you mean that the entire block should be placed somewhere else?

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (liangqigong) [reply portfolio/src/main/webapp/style.css:138]: Yes, I guess both. @hlz03 Please correct me if I am wrong.

https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

**Pull request merged**